indico/modules/events/surveys/util.py

```diff
@@ -59,3 +65,35 @@ def was_survey_submitted(survey):
     if submission_id is None:
         return False
     return bool(SurveySubmission.find(id=submission_id).count())
+
+
+class CSVSubmissionsExporter(object):
+
+    BASE_FIELDNAMES = {'submitter', 'submission_date'}
+
+    def __init__(self, survey):
+        self._survey = survey
+        self._fieldnames = self.BASE_FIELDNAMES | {question.title for question in self._survey.questions
+                                                   if not isinstance(question.field, StaticTextField)}
+
+    def generate(self):
+        with NamedTemporaryFile(suffix='indico.tmp', dir=Config.getInstance().getTempDir(), delete=False) as temp_file:
+            writer = csv.DictWriter(temp_file, fieldnames=self._fieldnames)
+            writer.writeheader()
+            for submission in self._survey.submissions:
+                submission_dict = {
+                    'submitter': (encode_if_unicode(to_unicode(submission.user.full_name))
+                                  if submission.user else 'Anonymous'),
+                    'submission_date': format_datetime(submission.submitted_dt),
+                }
+
+                for answer in submission.answers:
+                    submission_dict[answer.question.title] = _prepare_data(answer.answer_data)
+                writer.writerow(submission_dict)
+            return temp_file
+
+
+def _prepare_data(data):
+    if isinstance(data, list):
+        return encode_if_unicode('{}'.format(to_unicode(','.join(data))))
```

Comment:
that format call is unnecessary. you can simply remove it and will get the same behavior